### ISSUE TITLE
Add "Easy install" instructions paragraph in README.me, that works on most fresh-installed Linux (don't assume everyone knows npm, brew, etc.)

### ISSUE THREAD

**Issue by josephernest** [tldr-pages/tldr#1592]
I really love the project, I'm [not](https://github.com/josephernest/) a newbie in Linux or Windows, but still I'm a bit lost when trying to set up `tldr`, so I think I'm probably not the only one.

It seems that the README.me could be a little bit clearer about the mechanism. Is it:

1. Client (=your own computer) / Server (= one repo like Github pages), i.e. each time you do `tldr tar` on your computer, **it sends a request to internet** `https://raw.githubusercontent.com/tldr-pages/tldr/master/pages/common/tar.md` and displays the result on your computer

or 

2. Client (=your own computer) / Server (=your own computer), i.e. each time you do `tldr tar` on your computer, it sends a request to your own computer, then in this case, it should be obvious that we have to update ourself (`git pull` every now and then to be sure to have the latest version)

or

3. No client/server way. All is on your computer. Just a load of .md files, and `tldr tar` is just a shortcut to `less /.../pages/common/tar.md`.

**I think 1. or 3. would be the easiest for new users.**

Once again, I really like the wonderful idea of `tldr`, and I think **it deserves a one or two-lines install procedure** for users not so familiar with installing a node server or python or whatever server.

**Comment by agnivade** [comment]
It is 3. All files are stored locally. It is not just a shortcut though, the clients have to parse the page. We use a custom format to signify certain things. And also they color the output for beautification.

There is no mention of the word "server" anywhere. So, I am not sure where is the misunderstanding here. But if there is a confusion, please let us know what should be changed so that we can rectify it.

And it _is_ a one line install. The README mentions the install steps for all the clients.

**Comment by josephernest** [comment]
Thanks for your answer.

> So, I am not sure where is the misunderstanding here. 

This is the reason:

* I saw "client" (very often used in client/server situations, such as email clients (i.e. that connects to a mail server), web client / connected to a server)

* then I saw `Install it with sudo npm install -g tldr`, and I thought "that's Node", then I looked at the clients list, and saw `Node.js client : npm install -g tldr`, this was the confirmation in my head of the fact "it requires a Node.js server running". 

* then I understood it was 1. or 2.

> And it is a one line install. The README mentions the install steps for all the clients.

The problem for the new user of your tool when seeing this 20 or 30 rows list is "Which one should I choose?"

For example, **the first I tried** is this one: https://github.com/raylee/tldr. I followed the install steps in here: https://github.com/raylee/tldr#setup. Then I tried `tldr tar`. I got:

    tldr page for command tar not found

IMHO, there should a general easy one-line install solution for Linux (without requiring npm installed or such) on top of README.
So that the new user hasn't to think about "Which one of all these clients should I use"?

**Comment by agnivade** [comment]
> I saw "client" (very often used in client/server situations, such as email clients (i.e. that connects to a mail server), web client / connected to a server)

A "client" does not necessarily mean a client - server situation. By now, I think we have used "client" so ubiquitously, it will be too late to change it to anything else.

> then I saw Install it with sudo npm install -g tldr, and I thought "that's Node", then I looked at the clients list, and saw Node.js client : npm install -g tldr, this was the confirmation in my head of the fact "it will require having a Node.js server running".

`npm install -g tldr` does not mean in any way that a node.js server has to be running. It just install a couple of javascript files in your system.

Regarding which client to use, the 2nd line itself mentions to install the node client. So I think the choice should be pretty clear. If you don't want to use npm or any other dependencies, you are welcome to use the c++ client. Or feel free to comment on the other issue if you have any other ideas.

Regarding the error that you are getting, please open an issue in the tldr-node-client repo with all the details so that we can debug the issue.

**Comment by josephernest** [comment]
> A "client" does not necessarily mean a client - server situation. 

Yes of course, but when you see "clients" + you hear about node.js, it *can* make think of a client/server scheme.

> By now, I think we have used "client" so ubiquitously, it will be too late to change it to anything else.

That's right, I totally understand this.

> If you don't want to use npm or any other dependencies, you are welcome to use the c++ client.

What I mean is that nor `npm` nor `brew` are installed by default on a Debian or Ubuntu.

So maybe a case of [curse of knowledge](https://en.wikipedia.org/wiki/Curse_of_knowledge), but for a new user coming, he'll think:

Let's install it! `npm ...` ... hum it doesn't work (npm not installed by default). I should install npm first, but I don't know what it is. Let's try C++ version. `brew ...` hum it doesn't work (brew isn't installed by default), etc. Let's try the bash version. [I did and got an error]...
Hum, which other client to choose?

Once again I really find the tool excellent, don't misunderstand me, but I want to emphasize that, from the perspective of a new user, he might be lost in trying to install it, because your install instruction doesn't work on a fresh new Debian or Ubuntu.

That's one I really wanted to give my feeling: I love so much when Github projects offer a `Easy install` as a few lines that I just need to copy/paste and it will work on any fresh Debian/Ubuntu/or any other distro.

A final story: do you know how I heard about tldr from? From a friend who heard about it a few months ago ("it's a cool tool") but who finally didn't install it (maybe because he had the same difficulty).

**Comment by agnivade** [comment]
> because your install instruction doesn't work on a fresh new Linux.

Right. It would be great if we had something like that. But unfortunately we don't. You are most welcome to start and maintain a ppa for the C++ client. We will gladly add that.

Regarding your error with the bash client, feel free to open an issue in the bash client repo.

**Comment by sbrl** [comment]
Node / npm does not mean that a node server has to be running. In fact, a large number of packages available today are command-line applications that are set to 'prefer global', such that when you `npm install` them, they are immediately available in your path.

As for 'easy install' instructions, that's a tricky one. We could, for example, include instructions for installing node, like `sudo apt install nodejs` - but that would only work for debian-based machines. What about Arch, or even Windows and MacOS? It might be better then to link to the official node.js website saying (requires that [Node.JS](https://nodejs.org/) is installed).

**Comment by sbrl** [comment]
In fact, there's even an [online client](https://tldr.sh/) that we link to in the repository that yo  can use that requires no installation at all :P
